custom/app-setup/kafka/roles/configure-kafka/vars/main.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
can we take monitoring info/versions also here? We might also want to take litmus tunables (exporter version).